python/core/auto_generated/vector/qgsvectorlayertemporalproperties.sip.in

```diff
@@ -328,7 +328,7 @@ occur before or within the map's temporal range should be rendered).
 .. seealso:: :py:func:`accumulateFeatures`
 %End
 
-    QString createFilterString( const QgsVectorLayerTemporalContext &context, const QgsDateTimeRange &range ) const;
+    QString createFilterString( QgsVectorLayerTemporalContext context, const QgsDateTimeRange &range ) const;
```

Comment:
See other comment, but I think reading the code of this method would be reading better if 'range' was called 'filter_range' or so.

Reply:
I've changed this in the .cpp file, but I can't change the name in the header file without breaking API

Reply:
Ah, I mostly named the name of the parameter... I thought parameter names were not considered part of the api?

Reply:
They are, because changing them would break pyqgis code which uses:

    ... = createFilterString(..., range=...)

Reply:
Ouch, yes off course. Thanks for this insight.